docs/boilerplate/custom-fonts.md

```diff
@@ -0,0 +1,81 @@
+---
+id: boilerplate-custom-fonts
+title: Custom Fonts
```

Comment:
This is just a suggestion but maybe a more _questiony_ title? Like `How to use custom fonts`? What do you think?